app/modules/pangolin.nf

```diff
@@ -33,5 +33,13 @@ process PANGOLIN_PIPELINE {
       --outfile ${output_filename} \
       --datadir ${pangolin_data_dir} \
       --use-old-datadir
+
+
+    # Add pangolin_data_version column to the end of the result CSV file
+    declare -r pangolin_data_version=\$(pangolin --pangolin-data-version | cut -d ' ' -f 2)
```

Comment:
you'll need to pass --datadir here as well (like above)

Reply:
Added --datadir parameter.